plugins/free_space.js

```diff
@@ -18,7 +18,7 @@ exports.plugin = {
       
         let query = { custom_var_name: "freeSpaceInBytes" };
         let results = await db.collection(customVarsTable).findOneAndUpdate(query, { $set: { custom_var_value: diskSpace.free.toString() } }, { returnNewDocument: true });
-        server.publish('/ws/status/updateCustomVars', { id: results.value._id, custom_var_name: results.value.custom_var_name, custom_var_value: results.value.custom_var_value } );
+        server.publish('/ws/status/updateCustomVars', { id: results._id, custom_var_name: results.value.custom_var_name, custom_var_value: results.value.custom_var_value } );
```

Comment:
The database structure according to `db_custom_vars.js` is:

```js
      {
        _id: new ObjectID('59810167212b348aed7fa9f5'),
        custom_var_name: 'asnapStatus',
        custom_var_value: 'Off'
      },
```

[`findOneAndUpdate()`](https://www.mongodb.com/docs/manual/reference/method/db.collection.findOneAndUpdate/#examples) just returns the new document (because `returnNewDocument: true`). Where is `.value` coming from at all?

Reply:
Glad you caught this, and the story here is confusing. Older versions of `hapi-mongodb` (9.0.3 in our case) return the result wrapped in a "value" object. Newer versions (>=10) were not returning insertion counts, which are used throughout the codebase (spec makes me think it should have still returned counts, even for `insertOne()`, I'm unclear why the upgrade removed that).

9.0.3 appears to work fine with Node 20, however, so I decided to just revert to that version. I removed `value` here when I thought I was going to be using the upgraded version, and forgot to revert that. Just committed a fix.

Reply:
Worth the 5 minutes now to make the code conditional on whether `.value` is present in the object or not, so when we upgrade to `hapi-mongodb` in a year we're prepared for it? Or a comment at least?

Reply:
There are many, perhaps hundreds, of instances where `.value` is used today. That is my other reason for not bothering today. 🙃